osler/inventory/views.py

```diff
@@ -155,14 +155,21 @@ def export_csv(request):
         select_related('unit').\
         select_related('category').\
         select_related('manufacturer').\
-        order_by('category', 'name')
+        order_by('category', 'name', 'dose', 'expiration_date')
+    
+    day_interval = datetime.date.today() - datetime.timedelta(days=6)
+    recently_dispensed = models.DispenseHistory.objects.filter(written_datetime__gte=day_interval)
 
     with NamedTemporaryFile(mode='a+') as file:
         writer = csv.writer(file)
-        header = ['Drug Name', 'Dose', 'Unit', 'Stock', 'Expiration Date',
-                  'Lot Number', 'Category', 'Manufacturer']
+        header = ['Drug Name', 'Dosage', 'Unit', 'Remaining Stock', 'Expiration Date',
+                  'Lot Number', 'Category', 'Manufacturer', ''.join(['Doses Dispensed Since ', str(day_interval)])]
```

Comment:
Instead of using join, I think you can use f-strings to make the code look cleaner.